docs/quickstart/python.md

```diff
@@ -172,6 +174,13 @@ This regenerates `helloworld_pb2.py`, which contains our generated client and
 server classes, as well as classes for populating, serializing, and retrieving
 our request and response types.
 
+You could also have run
```

Comment:
(If we're leaving this "as is" and Nathaniel doesn't have other suggestions...)

Change to: If you want to call the code generation tool from the command line directly, you could run the following instead: